### PR TITLE
Replace fonts loaded via stylesheet with use of next/font

### PR DESCRIPTION
**Pull request by spencerwmiles**
Solves #1832 

I wasn't sure to use local font or just use google font. I can undo that change if use of Google font is undesirable

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@marsidev/react-turnstile": "^0.0.7",
     "@next-auth/prisma-adapter": "^1.0.5",
     "@next/bundle-analyzer": "^13.2.4",
-    "@next/font": "^13.2.4",
+    "@next/font": "13.0.6",
```
Comment (notmd) [inline website/package.json:40]: Is there any reason that you have pin the version?

**Review by notmd — COMMENTED** [review]
Thanks

File: website/package.json (hunk 1 above)
Comment (spencerwmiles) [reply website/package.json:40]: To match the version of `next` that is currently also using the same version (also pinned to 13.0.6)

The same was also done for `eslint-config-next` and `next` itself. The previously installed version was incompatible  and would result in error in dev and/or builds.

Comment (notmd) [reply website/package.json:40]: So we need to update next then, it's been long time since our last deps bump

**Review by notmd — APPROVED** [review]

**Pull request merged**